src/index.ts

```diff
@@ -5,22 +5,33 @@ import path from 'node:path'
 import { findRelease } from './release'
 import { checkerName, version } from './shared'
 
+const WORKING_DIR = path.join(process.env.HOME ?? "/tmp", 'editorconfig-checker')
```

Comment:
```suggestion
const WORKING_DIR = path.join(os.homedir(), 'editorconfig-checker')
```

https://nodejs.org/docs/latest/api/os.html#oshomedir

Reply:
Thank you 🙏

Reply:
Why using homedir instead of tmpdir?

I'm fine with the current change, because I don't understand it fully, but I'm asking to be sure